### DISCUSSION TITLE
Need help receiving RTMP stream from IP camera

### DISCUSSION THREAD

**Discussion by usa-** [bluenviron/mediamtx#3613] [Questions]
### Question

On my Raspberry PI server, I've just compiled mediamtx with default parameters and run the executable.
The server's IP address is 192.168.0.2.
I've got [this](https://www.aliexpress.com/item/1005004626674100.html) IP camera.
In the camera's settings, I've created a RTMP push connection to rtmp://192.168.0.2/mystream.
mediamtx writes:
```
2024/08/04 18:55:00 INF [RTMP] [conn 192.168.1.188:42851] opened
2024/08/04 18:55:00 INF [RTMP] [conn 192.168.1.188:42851] closed: received type 1 chunk without previous chunk
2024/08/04 18:55:10 INF [RTMP] [conn 192.168.1.188:42852] opened
2024/08/04 18:55:10 INF [RTMP] [conn 192.168.1.188:42852] closed: received type 1 chunk without previous chunk
```
What's wrong?

**Comment by aler9** [comment]
This is impossible to replicate without a network dump. Open a bug report and follow instructions to generate a network dump that allows to replicate the issue.

**Reply by usa-** [reply]
Sorry, the problem was with the publishing application. It has been fixed now.

[a comment by an automated account is not shown]
